spec/dummy/app/views/components/modal.slim

```diff
@@ -0,0 +1,6 @@
+button#show Show
+
+= ui_component 'modal', \
```

Comment:
Or we just stick with the [bootstrap way of doing it](http://getbootstrap.com/javascript/#live-demo).